apps/cugetreg-web/src/common/components/CookieBanner/index.tsx

```diff
@@ -53,7 +53,7 @@ export const CookieBanner = () => {
             <Typography variant="body1" mb={2}>
               เราใช้คุกกี้เพื่อพัฒนาประสิทธิภาพ และประสบการณ์ที่ดีในการใช้เว็บไซต์ของคุณ
               คุณสามารถศึกษารายละเอียดได้ที่{' '}
-              <Link href="/privacy" passHref>
+              <Link href="/privacy" passHref legacyBehavior>
```

Comment:
what exactly does this do?

Reply:
New `next/link` no longer required children `<a>` and if you put that it will cause an error.
We use custom components which have `<a>` here so we need `legacyBehavior` to retain the previous behavior

https://nextjs.org/docs/api-reference/next/link